### DISCUSSION TITLE
✅ After the update

### DISCUSSION THREAD

**Discussion by taygunis** [isontheline/pro.webssh.net#1031] [Ideas]
Hello i am using your program on my macbook and i was loving it but after the update i cannot find cloning from ssh to sftp as easy as before. 
And another thing is a suggestion when i try to copy data from server to my pc im doing it one by one and i cannot do it with files. Other than that its almost perfect tool for me.

**Comment by isontheline** [comment]
Hello @taygunis 👋

Thank you so much for your feedback and your support about WebSSH 🙏

Cloning is no more available because you can now set multiple roles (SSH / SFTP) to the same connection. Just edit your connection and add roles on it.

You're right about the multiple download files, an old issue is here : https://github.com/isontheline/pro.webssh.net/issues/205
I need to work on it but I can't tell any ETA yet. I will try to do my best.

Many thanks again!

**Reply by taygunis** [reply]
that helps alot thank you. you are the best 👍
